optimize/util/optimize-commons/pom.xml

```diff
@@ -16,13 +16,13 @@
     <dependency>
       <groupId>io.camunda</groupId>
       <artifactId>camunda-search-client-connect</artifactId>
-      <version>8.6.5</version>
+      <version>8.6.10</version>
     </dependency>
 
     <dependency>
       <groupId>io.camunda</groupId>
       <artifactId>camunda-search-client-plugin</artifactId>
-      <version>8.6.5</version>
+      <version>8.6.10</version>
```

Comment:
Didn't I already change this yesterday?

Reply:
This PR merges all the changes of the release branch back to the stable branch

Reply:
I am doing this:
https://bru-2.tasklist.camunda.io/8da2f67e-88ab-4b38-9e39-52bb0ea4855f/tasklist/4503599966239900